sintr_analysis_server_example/bin/decompress.dart

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:path/path.dart';
+import 'package:sintr_worker_lib/instrumentation_transformer.dart';
+
+/// Decompress a compressed instrumentation file.
+main(List<String> args) async {
+  if (args.length != 1) {
+    print('Usage: decompress filePath');
+    exit(1);
+  }
+
+  // Initialization
+  final path = args[0];
+  var srcName = basename(path);
+  if (!srcName.startsWith('compressed-')) throw 'file is already decompressed';
+  var srcFile = new File(path);
+  if (!srcFile.existsSync()) throw 'cannot find $path';
+  var dstName = srcName.substring(11);
```

Comment:
Could we do dstName = "${srcName}.unpacked" instead. It's safer to add things to file names than remove things.